config.m4

```diff
@@ -402,6 +402,10 @@ PHP_ARG_WITH(libmongoc, whether to use system libmongoc,
     AC_SUBST(MONGOC_ENABLE_SECURE_TRANSPORT, 0)
     AC_SUBST(MONGOC_ENABLE_COMMON_CRYPTO, 0)
 
+    dnl Secure Channel only applies to Windows
+    AC_SUBST(MONGOC_ENABLE_SECURE_CHANNEL, 0)
```

Comment:
Apologize upfront. This will be renamed very soon.

MONGOC_ENABLE_SSL_SECURE_CHANNEL
MONGOC_ENABLE_SSL_SECURE_TRANSPORT
MONGOC_ENABLE_SSL_OPENSSL

same with crypto
MONGOC_ENABLE_CRYPTO_COMMON_CRYPTO
MONGOC_ENABLE_CRYPTO_CNG
MONGOC_ENABLE_CRYPTO_LIBCRYPTO